Fabric.Authorization.AccessControl/src/app/services/interceptors/fabric-http-fake-discovery-interceptor.service.ts

```diff
@@ -0,0 +1,44 @@
+import { Injectable } from '@angular/core';
+import { Observable, of, throwError } from 'rxjs';
+import {
+  HttpInterceptor,
+  HttpRequest,
+  HttpResponse,
+  HttpHandler,
+  HttpEvent
+} from '@angular/common/http';
+import { ConfigService } from '../global/config.service';
+
+@Injectable()
+export class FabricHttpFakeDiscoveryInterceptorService implements HttpInterceptor {
+  private responseBody : any = null;
+
+  constructor(private configService: ConfigService) { }
+
+  intercept(
+    req: HttpRequest<any>,
+    next: HttpHandler
+  ): Observable<HttpEvent<any>> {
+    if(req.url.includes('DiscoveryService/v1/Services?$filter=ServiceName')){
+        this.configService.getIdentityServiceRoot().subscribe(url => {
+          this.responseBody = {
+            "@odata.context":"http://localhost/DiscoveryService/v1/$metadata#Services(ServiceUrl,Version,ServiceName)","value":[
+              {
+                "ServiceUrl":"http://localhost/IdentityProviderSearchService/v1","Version":1,"ServiceName":"IdentityProviderSearchService"
+              },{
+                "ServiceUrl":url,"Version":1,"ServiceName":"IdentityService"
+              },{
+                "ServiceUrl":"http://localhost/AuthorizationDev/v1","Version":1,"ServiceName":"AuthorizationService"
```

Comment:
Is the localhost going to cause problems if everything else is registered with domain name?  Maybe all you need from discovery service is IdPSS

Reply:
The key here is that we are using a fully qualified url for interacting with identity, which will come from appsettings.json. Everything else should work correctly from then on.